.github/workflows/rust.yml

```diff
@@ -313,7 +313,17 @@ jobs:
       - name: Read results
         if: always()
         run: |
-          find ${{ matrix.directory }}/${{ env.OUTPUT_DIRECTORY }} -name timing.json -exec cat {} + | jq '{(.test_path): . | del(.test_path)}' | jq -s add
+          message=$(find ${{ matrix.directory }}/${{ env.OUTPUT_DIRECTORY }} -name timing.json -exec cat {} + | jq '{(.test_path): . | del(.test_path)}' | jq -s add)
+          directory="${{ matrix.directory }}"
+          package="${directory##*/}"
+          tags="package:$package"
+          curl -X POST "https://http-intake.logs.datadoghq.com/api/v2/logs" \
+            -H "Content-Type: application/json" \
+            -H "DD-API-KEY: ${{ secrets.DATADOG_API_KEY }}" \
+            -d $(jq -n \
+                    --argjson message "$message" \
+                    --arg ddtags "$tags" \
+                    '{ message: $message, ddsource: "hash-gh-actions", hostname: "github", service: "actions", ddtags: $ddtags }')
```

Comment:
Could you push one with this format manually with curl (although maybe change one or two of the keys so they don't appear later on, maybe add test- in front of the source or something)

And then post the example in a screenshot in the PR?